tests/passes/static-timing/compile-seq-mixed.futil

```diff
@@ -0,0 +1,48 @@
+// -x top-down-st:dump-fsm -p top-down-st
```

Comment:
Probably should have `-b none` so that the above expect file doesn't have the output Calyx file code.